azure-functions/src/send_grid/message_builder.rs

```diff
@@ -19,8 +19,13 @@ impl MessageBuilder {
     /// # Examples
     ///
     /// ```rust
-    /// # use azure_functions::send_grid::MessageBuilder;
-    /// let message = MessageBuilder::new().to("foo@example.com").build();
+    // @peterhuene, I removed a `#` from this example that I think was included by mistake.
```

Comment:
The `#` in a doc test is treated as a statement to the compiler and hides it from the example.  Generally I try to hide the `use` statement in the examples since the documentation makes clear which module the type is in.  So we can probably add it back.